doc/interfaces/index.rst

```diff
@@ -9,4 +9,5 @@ DIPY Workflows Interfaces
     basic_flow.rst
     buan_flow.rst
     bundle_segmentation_flow.rst
-    data_fetch.rst
\ No newline at end of file
+    data_fetch.rst
+    denoise.rst
```

Comment:
Hi @areeshatariq, the order here is important to help the user. Can you update with the following order:

    basic_flow.rst
    data_fetch.rst
    denoise.rst
    buan_flow.rst
    bundle_segmentation_flow.rst

Furthermore, Can you update both filenames by appending `*_flow.rst`.

Thank you